src/blocked.c

```diff
@@ -563,6 +564,7 @@ void serveClientsBlockedOnKeyByModule(readyList *rl) {
  * do, the function is already fair. */
 void handleClientsBlockedOnKeys(void) {
     server.core_propagates = 1;
+    server.propagate_no_wrap = 1;
```

Comment:
before this commit, we didn't have MULTI/EXEC and I was under the impression that we want to avoid unnecessary transactions.

if we do want to inject a redundant MULTI/EXEC I think it should be around all the commands propagated by handleClientsBlockedOnKeys, I don't see any reason to group the MULTI/EXECs by type (list,zset,stream,module)

Reply:
It's not by type, it'd by command. 
Ie. BLMOVE or some other blocked command can propagate two commands, and these should be in transaction. But the entire event of processing all unblocked clients as a transaction is wrong (they're unrelated operations)